src/historywork/ApplyLedgerChainWork.h

```diff
@@ -4,9 +4,9 @@
 
 #pragma once
 
+#include "work/Work.h"
 #include "herder/TxSetFrame.h"
```

Comment:
this will get reverted by clang-format, maye add an empty line after work/Work.h?

Reply:
I thought I ran clang-format... but I didn't. Thanks for spotting this.